lib/auth/legacy.js

```diff
@@ -14,6 +15,10 @@ const loginPrompter = async (creds) => {
 const login = async (npm, opts) => {
   let res
 
+  if (opts.authType === 'legacy') {
```

Comment:
I'm not sure why this if statement is here.  If it's not, then this file wouldn't be used.

Reply:
Oh I see `web` points here.